### PR TITLE
added models

### PR DESCRIPTION
**Pull request by volivers**
@BoavidaGuerra can you review the models? many thanks 😎

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,6 +4,12 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :username, presence: true
+  has_many :bookings, dependent: :destroy
+  has_many :skills
```
Comment (BoavidaGuerra) [inline app/models/user.rb]: add `,dependent: :destroy`

Comment (BoavidaGuerra) [inline app/models/user.rb]: add `,dependent: :destroy`

**Review by BoavidaGuerra — CHANGES_REQUESTED** [review]
Add "dependent: :destroy"

**Review by volivers — COMMENTED** [review]
User model updated :)

**Review by BoavidaGuerra — APPROVED** [review]

**Pull request merged**